modules/core/perf/perf_reduce.cpp

```diff
@@ -65,4 +65,33 @@ PERF_TEST_P(Size_MatType_ROp, reduceC,
     SANITY_CHECK(vec, 1);
 }
 
+typedef tuple<Size, MatType, int> Size_MatType_RMode_t;
+typedef perf::TestBaseWithParam<Size_MatType_RMode_t> Size_MatType_RMode;
+
+PERF_TEST_P(Size_MatType_RMode, reduceArgMinMax, testing::Combine(
```

Comment:
Do we really need performance tests now? Can we disable them by default (`DISABLED_reduceArgMinMax`)?

- what is about 3D / 4D layouts? (DNN doesn't use 2D matrices in general)

Reply:
3D/4D layouts perform similarly if the number of pixels is the same.